modules/mobile/spec/request/facilities_info_request_spec.rb

```diff
@@ -153,5 +153,36 @@
         end
       end
     end
+
+    context 'when there are no appointments' do
+      before do
+        Mobile::V0::Appointment.set_cached(user, [])
+      end
+
+      it 'returns facility details sorted alphabetically' do
+        VCR.use_cassette('appointments/get_multiple_mfs_facilities_200', match_requests_on: %i[method uri]) do
+          get '/mobile/v0/facilities-info/appointments', headers: iam_headers, params: params
+          facilities = response.parsed_body.dig('data', 'attributes', 'facilities')
+          expect(response).to have_http_status(:ok)
+          expect(facilities[0]['id']).to eq('358')
```

Comment:
Can we test by facility names instead of ids to demonstrate the alphabetical-ness?

Reply:
Agreed.

Reply:
Sure, I just copied an existing spec for testing alphabetical-ness since those were already "approved". Should I go ahead and change those as well?

Reply:
Based on the conversation here, I'm realizing that the comment probably is wrong and the specs probably are correct. When there are no previous appointments for a facility, it just sorts by id. It makes me wonder if what we really wanted was to sort by name here as a fallback instead of id, like:
`facilities.sort_by { |facility| [rf_hash[facility.id] || recent_facilities.size, facility.name] }`